test/contracts/Vouching.test.js

```diff
@@ -3906,6 +3914,44 @@ contract('Vouching', function (accounts) {
     })
   })
 
+  describe('transferAppealsResolution', function () {
```

Comment:
I have lost track of how many times we've had to write this code with slight variations... :-P

Reply:
couldn't agree more